application/search/filters.py

```
@@ -280,18 +280,44 @@ class QueryFilters:
+                    geometry_str = str(geometry).strip()
+                    if geometry_str.startswith('{"type"') or geometry_str.startswith(
+                        '"{"type"'
+                    ):
+                        raise InvalidGeometry(
+                            "Invalid geometry format. Expected WKT format "
+                            "(e.g., 'POLYGON((...))'), but received GeoJSON format. "
+                            "(e.g., '{'type': 'Polygon', 'coordinates': [[...]]}'. "
+                            "Please convert your GeoJSON to WKT format."
+                        )
+                    else:
+                        raise InvalidGeometry(f"Invalid geometry {geometry}")
```

Pylint (4.0.5)

[refactor] Unnecessary "else" after "raise", remove the "else" and de-indent the code inside it

(R1720)